### PR TITLE
Update DG and UG

### LINKED ISSUE #238: Payment date Future Dating validation should be stated on UG
Currently it does not say that the last payment date cannot be in the future (although it is common sense). 
This causes an error when the user pastes the first example from the UG, since the payment date in the example is 12/12/20.
`add n/Alex p/93211234 s/Commonwealth Secondary School y/pri 6 v/Blk 33 West Coast Rd #21-214 t/1 1430-1630 f/25 d/12/12/2020`
In the testing phase, users might also flag up this as a bug (since an error is shown), so i believe we should change the example to ensure they dont.

### LINKED ISSUE #241: Inconsistency in displaying academic details
Exams are displayed in a inconsistency manner compared to questions and attendances

### PR DESCRIPTION
**Pull request by hogantan**
Add (UG):
* UG images for add, add exam, delete exam, exam stats and toggle command

Add command:
AddCommandExpectedOutcomeUG

Add exam command:
AddExamCommandExpectedOutcomeUG

Delete exam command:
DeleteExamExpetedOutcomeBefore
DeleteExamExpetedOutcomeAfter

Exam stats command:
ExamStatsCommandExpectedOutcomeUG

Toggle command:
ToggleCommandExpectedOutcomeBeforeUG
ToggleCommandExpectedOutcomeAfterUG

Add (DG):
* Design considerations for toggle and exam stats command 

Update:
* toString() of Academic and Exam to standardize how academic fields are displayed in GUI 

Issues:
#239 
#240 
#241
#243

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -638,8 +720,13 @@ The following table provides the definitions of the various terms used in this U
 
 Term | Definition
 --------|------------------
+Private Tutor | A person that teaches students in a 1 to 1 manner.
+Student | A person that is being taught by a private tutor.
```
Comment (ong-yinggao98) [inline docs/UserGuide.md]: As our target audience are private tutors, we probably wouldn't need a glossary for these 2.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -210,12 +227,23 @@ Format: `add n/NAME p/PHONE s/SCHOOL y/YEAR v/CLASS_VENUE t/CLASS_TIME [f/FEE] [
 
 </div>
 
+<div markdown="block" class="alert alert-info">
+
+:warning: `LAST_PAYMENT_DATE` cannot be a future date (e.g. cannot be later than the current date)
```
Comment (ong-yinggao98) [inline docs/UserGuide.md]: Not sure if this was my fault, but could you change the e.g. to i.e.?

**Review by ong-yinggao98 — COMMENTED** [review]
Aside from the two I raised, you might want to consider toning down the capitalisation of the annotations in your screenshots and the glossary, it seems a bit jarring to see "Displaying Academic Details of Student" in red when it could be "Displaying academic details of student". Think the vibe that full caps gives.

File: docs/UserGuide.md (hunk 1 above)
Comment (AlexCQY) [reply docs/UserGuide.md]: I think we could just put it there for greater clarity. Though I completely understand that these could be implied.

**Review by AlexCQY — COMMENTED** [review]
Do you plan to change the Ui you have added to the latest one?

**Review by ong-yinggao98 — APPROVED** [review]
LTGM

**Pull request merged**